### ISSUE TITLE
following dockerhub 'Running with Docker' instructions gives error 404

### ISSUE THREAD

**Issue by mi-hol** [brettt89/silverstripe-web#29]
Appreciate the work you put into documenting the Howto of using  SilverStripe and containers!

Being a newcomer to both, SilverStripe and containers, I've tried to learn from your instruction but failed.  Most likely I missed or misunderstood a file or preparation step obvious to all experts.

on https://hub.docker.com/r/brettt89/silverstripe-web/ section 'Running with Docker'
says:
```
This image can also be run directly with docker. However it will need to be linked with a database in order for SilverStripe to successfully build.

docker run -p 3306:3306 --name database mysql
docker run -p 80:80 -v /path/to/project:/var/www/html --link database --name project1 brettt89/silverstripe-web
NOTE: You will require an _ss_environment.php file to tell the environment which database to connect to. An example one has been provided in ./example folder example

You should then be able to access run a dev buid via http://localhost/dev/build.
```
1) error from mysql:
error: database is uninitialized and password option is not specified
  You need to specify one of MYSQL_ROOT_PASSWORD, MYSQL_ALLOW_EMPTY_PASSWORD and MYSQL_RANDOM_ROOT_PASSWORD

fixed by adding environment variable & --detach
```
docker run -p 3306:3306 --name database  --detach --env MYSQL_ALLOW_EMPTY_PASSWORD=1 mysql
```

as port 80 was alreayd used I changed to port 81 but get now 
2. error 404 from  http://localhost:81/dev/build
```
docker ps
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS                               NAMES
575afcaa398b        brettt89/silverstripe-web   "docker-php-entrypoi…"   2 hours ago         Up 2 hours          0.0.0.0:81->80/tcp                  project1
5f07b616daa2        mysql                       "docker-entrypoint.s…"   2 hours ago         Up 2 hours          0.0.0.0:3306->3306/tcp, 33060/tcp   database
```
the files I have in ' /path/to/project' are:

```
Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----       29.03.2019     13:58                moderno-admin
d-----       03.04.2019     11:11                mysite
-a----       10.04.2019     13:39            683 _ss_environment.php
```
_ss_environment.php was downloaded from https://github.com/brettt89/silverstripe-web/blob/master/examples/_ss_environment.php

Any hint greatly appreciated

**Comment by brettt89** [comment]
Hi @mi-hol Apologies for the delay in responses. Have been on a holiday for a bit.

The 404 would indicate that Apache is unable to find the root files of the website. I noticed you provided a file structure listing of your website. This appears to only contain 2 folders (mysite, moderno-admin).

I would expect this directory to also contain an `index.php` file as per SilverStripe installations (Web root).

How have you installed SilverStripe? Did you use composer?

**Comment by mi-hol** [comment]
Hi Brett,
a late answer is much better than none at all :)

Unfortunately I don't have the setup used anymore.
But will try again shortly.

**Comment by mi-hol** [comment]
regarding the issue you wrote 'I would expect this directory to also contain an index.php file as per SilverStripe installations (Web root)'  
on https://hub.docker.com/r/brettt89/silverstripe-web/ section 'Running with Docker' neither  mentions 'a Silverstrip installation' nor 'index.php' as a prerequisite, hence I'd believe that is the root of confusion.
Before I can use the container, what are the steps I'd need to execute (link to doc would be perfect)?

**Comment by patricknelson** [comment]
Unless I'm mistaken, this doesn't actually come with Silverstripe installed, correct? To help get you started though I think you'll still need to SSH into the container and `composer install` all the things, e.g. in your case (and I'm **totally** winging it here based on https://github.com/brettt89/silverstripe-web/blob/e55cf50f274caa314974b6ce775cb14efc95743d/.travis.yml#L65), try running:

```
# Login to container (drop into a bash shell)
docker exec -it project1 bash

# Switch into project root (if not already) and install SS project here.
# NOTE: Ensure you've already got a shared volume mount to /var/www/html otherwise you may lose your installation and start over again
cd /var/www/html
composer create-project silverstripe/installer ~src 4 
```

Maybe that'll kick ya off into the right direction 😄

**Comment by brettt89** [comment]
I have updated the `README.md` to provide more instructions around using this container. This includes an optional installation of SilverStripe.

Fixed by: #35
